contracts/test/integration/ArbToEth.ts

```
@@ -1117,3 +1254,75 @@ describe("Integration tests", async () => {
+    honest: 0,
+    challenger: ethers.ZeroAddress,
+  });
+}
+
+export async function decodeMessage(msg: any) {
+  const nonce = "0x" + msg.slice(2, 18);
+  const to = "0x" + msg.slice(18, 58); //18+40
+  const from = "0x" + msg.slice(58, 98); //58+40
+  const msgData = "0x" + msg.slice(98);
+  return { nonce, to, from, msgData };
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)